lib/api/index.js

```diff
@@ -1,4 +1,18 @@
+'use strict'
+
+/**
+ * Handles a request to a not implemented `/api/` endpoint.
+ * @param req
+ * @param res
+ * @param next
+ */
+function notImplemented (req, res, next) {
+  res.status(404).send('API not implemented')
+}
+
 module.exports = {
-  accounts: require('./accounts'),
-  messages: require('./messages')
+  authn: require('./authn'),
```

Comment:
In this case I can see that as being reasonable. I think it would be nice if the namespacing in the exports object reflect the directory structure of the API modules themselves in that case to avoid confusion.